src/jetson/AES.py

```diff
@@ -8,23 +8,39 @@
 THRES = 400
```

Comment:
Please provide a docstring explaining this constant. I understand its a threshold but I don't understand what it is a threshold for or what units it is in.